.github/workflows/black.yml

```diff
@@ -0,0 +1,50 @@
+name: Black
+
+on:
+  workflow_dispatch: # Activate this workflow manually
+  pull_request:
+
+jobs:
+
+  check:
+    runs-on: ubuntu-latest
+    steps:
+
+      - uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: ./.github/actions/python_cache/
+
+      - name: Install Haystack
+        run: |
+          pip install --upgrade pip
+          pip install .[dev]
+
+      - name: Black
+        run: black .
+
+      - name: Check status
+        run: |
+          if [[ `git status --porcelain` ]]; then
```

Comment:
optional: you could also use `black --check` which returns an non-zero exit code instead of relying on git.

Reply:
ooh that's cool! Will do :+1: